alpine/entry.sh

```diff
@@ -24,15 +24,35 @@ mount --move /tmp /dev
 ln -sf /dev/pts/ptmx /dev/ptmx
 
 mount -t debugfs nodev /sys/kernel/debug
-udevd & 
-udevadm trigger &> /dev/null
+
+if [ "$INITSYSTEM" = "on" ]; then
+	GREEN='\033[0;32m'
+	echo -e "${GREEN}OpenRC init system enabled."
+	env >> /etc/docker.env
+	printf '#!/bin/bash\n' > /etc/resinApp.sh
+	printf '%q ' "$@" >> /etc/resinApp.sh
+	chmod +x /etc/resinApp.sh
+
+	touch /var/log/resinapp.log
 	
-CMD=$(which $1)
-# echo error message, when executable file doesn't exist.
-if [  $? == '0' ]; then
-	shift
-	exec "$CMD" "$@"
+	sed -i -e s~#{DIR}~"$(pwd)"~g \
+		-e s~#{ENV}~"$(env | sed '/PATH=/d'  | tr '\n' ' ' | xargs printf '--env %s ')"~g /etc/init.d/resin
+
+	rc sysinit --quiet
+	rc default 
+	tail -f /var/log/resinapp.log
```

Comment:
@petrosagg OpenRC won't start in a container where it's not pid 1
```
/sbin/init
init: must be run as PID 1
```
So not like systemd, we can't make it PID 1. However, we can replicate the init process by run openrc at `sysinit` runlevel to initialize the system and start services as we want.
Our service will always clean up all processes on stop or restart so there will be no orphaned processes that will potentially break the service